conans/client/packager.py

```diff
@@ -46,23 +48,26 @@ def create_package(conanfile, source_folder, build_folder, package_folder, insta
         conanfile.install_folder = install_folder
         conanfile.build_folder = build_folder
 
-        def recipe_has(conanfile, attribute):
+        def recipe_has(attribute):
             return attribute in conanfile.__class__.__dict__
 
         if source_folder != build_folder:
             conanfile.copy = FileCopier(source_folder, package_folder, build_folder)
             with conanfile_exception_formatter(str(conanfile), "package"):
                 with tools.chdir(source_folder):
                     conanfile.package()
-            warn = recipe_has(conanfile, "package")
-            conanfile.copy.report(package_output, warn=warn)
+            if (not conanfile.copy.report(package_output) and
```

Comment:
Use a line to call the ``report()`` method, otherwise it doesn't look like it is actually reporting things.